### ISSUE TITLE
Error: Failed to load nam model. Message:  Config JSON doesn't exist!

### ISSUE THREAD

**Issue by X-Raym** [sdatkinson/NeuralAmpModelerPlugin#120]
**Describe the bug**
Hi, it would be nice to have online steps by steps help for the error
"Failed to load nam model. Message:

Config JSON doesn't exist!"

![img](https://i.imgur.com/br9McIY.png)

It do link to [nam model repo](https://github.com/sdatkinson/nam-model-utility/) but it doesnt give more infos.
Also, this looks pretty similar to cvt2sgl.py found in https://github.com/pelennor2170/NAM_models/blob/main/cvt2sgl.py
but this also returns an error:

```
Traceback (most recent call last):
  File "E:\Téléchargements\NAM_models-main\cvt2sgl.py", line 19, in <module>
    with open(configFN, "r") as fr:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\USER\\3D Objects\\config.json'
```

So, better step by step resolution guide for this will helps for sure.

This makes me think this is for developers only in Python field, maybe not yet for main public?

Thx for your support!

**To Reproduce**
2. Click on Load Model
3. Scroll down to your Model

**Desktop (please complete the following information):**
 - OS Win10x64
 - Plugin version [e.g. 0.7.0]
 - VST3
 - REAPER

**Comment by X-Raym** [comment]
@sdatkinson ```E:\Téléchargements\NAM_models-main\George B Ceriatone King Kong KK Ch2 60s solo 80s.nam``` for eg. I tested various models and non worked.

**Comment by sdatkinson** [comment]
Thanks!

Does it work if you move them from `Téléchargements` to a location without any `é`s?

**Comment by X-Raym** [comment]
@sdatkinson It works indeed!

Funny, who would think that at this age of AI computation, having unicode character in a path was still a issue to look for 😅

Thx for the support, and, I guess, future update!

**Comment by sdatkinson** [comment]
Great! This is fixable 😅

Thanks!

**Comment by sdatkinson** [comment]
Closing--was fixed in v0.7.2
